build.gradle

```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.1.51'
+    ext.kotlin_version = '1.1.60'
```

Comment:
If you change this, you might as well bump to 2.0.

Reply:
Souns like a bigger bump that wareants a separate pr tbh. :)

Reply:
1.2.... The previous comment never happened....

Reply:
Trust AS
😁